### PR TITLE
New Packages: Othman and amiri-font

### PR DESCRIPTION
**Pull request by Johnnynator**

### CODE REVIEW

File: srcpkgs/amiri-font/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'amiri-font'
+pkgname=amiri-font
+version=0.111
+revision=1
+wrksrc="Amiri-${version}"
+hostmakedepends="unzip"
+short_desc="Classical Arabic typeface in Naskh style"
+maintainer="John <johnz@posteo.net>"
+license="OFL-1.1."
+#changelog="https://github.com/alif-type/amiri/blob/master/NEWS.md"
```
Comment (maxice8) [inline srcpkgs/amiri-font/template]: changeog doesn't need to be quoted, preferred location is below homepage=

**Comment by Johnnynator** [comment]
OK fixed it

File: srcpkgs/amiri-font/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'amiri-font'
+pkgname=amiri-font
+version=0.111
+revision=1
+wrksrc="Amiri-${version}"
+hostmakedepends="unzip"
+short_desc="Classical Arabic typeface in Naskh style"
+maintainer="John <johnz@posteo.net>"
+license="OFL-1.1."
```
[the remark this answers is not in the document]
Comment (Johnnynator) [reply srcpkgs/amiri-font/template]: removed the dot

File: srcpkgs/othman/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+version=0.5.1
+revision=1
+build_style=python2-module
+hostmakedepends="python-setuptools ImageMagick sqlite intltool"
+depends="python-sqlite python-gobject amiri-font"
+short_desc="Quran browser and search engine "
+maintainer="John <johnz@posteo.net>"
+license="WAQFv2"
+homepage="https://github.com/ojuba-org/othman/"
+distfiles="https://github.com/ojuba-org/othman/archive/0.5.1.tar.gz"
```
Comment (maxice8) [inline srcpkgs/othman/template]: should be ${version}

Comment (Johnnynator) [reply srcpkgs/othman/template]: fixed.

**Comment by asergi** [comment]
@maxice8 @Johnnynator at first glance (about othman):
- `noarch=yes` is missing
- `pycompile_module="othman"` is missing
- `setuptools` is not required
- the sed on the shebang is unnecessary: sed/patch the Makefile to make it use `python2` instead (i.e.: `python2 gen-index.py`)

**Comment by asergi** [comment]
It would be nice to stop adding `python2-only` packages.

**Comment by asergi** [comment]
amiri-font:
- `noarch=yes`
- `depends="font-util"`
- `vinstall $_i 644 usr/share/fonts/TTF`
- `font_dirs="/usr/share/fonts/TTF"`

**Pull request merged**